ADOPTERS.md

```diff
@@ -11,3 +11,6 @@ Example entry:
 -->
 
 This is a list of adopters of using PowerShell in production or in their products (in alphabetical order):
+
+  * [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
```

Comment:
Looks like I read the Codacy issue wrong, the bullet is left aligned, the content underneath is whitespaced
```suggestion
* [Azure Cloud Shell](https://shell.azure.com/) provides a batteries-included browser-based PowerShell environment used by Azure administrators to manage their environment.
```